packages/pds/tests/app-passwords.test.ts

```diff
@@ -0,0 +1,103 @@
+import AtpAgent from '@atproto/api'
```

Comment:
One other case I can think of that might be nice in here is is that tokens received after a refresh continue to be restricted as app-password credentials.

Reply:
ah good call 👍